.gitignore

```diff
@@ -1,54 +0,0 @@
-# standard python ignores
```

Comment:
Why are you deleting this file?

Reply:
That was an accident. Something to do with git clean and rebuilding docs I think.